src/test/groovy/ratpack/zipkin/support/TestReporter.groovy

```diff
@@ -3,18 +3,19 @@ package ratpack.zipkin.support
 import zipkin.Span
 import zipkin.reporter.Reporter
 
+import java.util.concurrent.ConcurrentLinkedDeque
 
 class TestReporter implements Reporter<Span> {
 
-	private final List<Span> spans = new LinkedList<>()
+	private final ConcurrentLinkedDeque<Span> spans = new ConcurrentLinkedDeque<>()
```

Comment:
Yeah just for thread-safety.  Same pattern used by the Brave integration tests.

Right now most of the tests are single threaded with the exception of the ParallelBatch tests so probably not necessary but it feels cleaner to have it :)